tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -287,6 +287,33 @@ crm/test_crm.py::test_crm_fdb_entry:
     conditions:
       - "'t0' not in topo_name and topo_type not in ['m0', 'mx']"
 
+#######################################
+#####           dash              #####
+#######################################
+dash/crm/test_dash_crm.py:
+  skip:
+    reason: "Currently dash tests are not supported on KVM"
+    conditions:
+      - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
+
+dash/test_dash_acl.py:
+  skip:
+    reason: "Currently dash tests are not supported on KVM"
+    conditions:
+      - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
+
+dash/test_dash_disable_enable_eni.py:
+  skip:
+    reason: "Currently dash tests are not supported on KVM"
+    conditions:
+      - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
+
+dash/test_relaxed_match_negative.py:
+  skip:
+    reason: "Currently dash tests are not supported on KVM"
+    conditions:
+      - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/16407"
+
```

Comment:
I agree with Yutong that a single entry for `dash` should be enough. No need to have so many entries.

Reply:
There is a script is in PR test scope for dpu platform

Reply:
I see, scripts under the `dash` folder are more than what listed here. The other scripts need to be covered and should not be skipped.

Reply:
That's what I want to ask, if script `dash/test_dash_vnet.py` really run on kvm or skip? If it is also skipped on kvm, we can use single entry to skip all of them. But if not, we can add another enrty for `dash/test_dash_vnet.py` to run it.